woocommerce/woocommerce
# [Product Block editor]: rename component edit function

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



This PR renames the edit function of a few components to make them easy to inspect when using the React dev tool.
Currently, all block defines their edit component function with the same `Edit` name.

### Changes proposed in this Pull Request:





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the Product Editor
2. Open the React dev tool
3. Search the block instances by typing `Tab`, `Section`, and `Description`.
4. Confirm you can select the block instances and inspect their properties, states, etc.

<img width="827" alt="Screenshot 2023-11-28 at 10 02 29" src="https://github.com/woocommerce/woocommerce/assets/77539/4533c87f-b65d-4bdc-bb34-390df257a756">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
